pkg/cmd/cli/backup/create_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 the Velero contributors.
+Copyright 2019, 2020 the Velero contributors.
```

Comment:
This should actually only be the latest year, not a comma separated list. We don't actually have this documented anywhere though, so I don't think this is a blocker.

Reply:
Yeah, I didn't see any doc related to this but I saw a few files have this format "2017, 2019" so I followed that style.  In my company, we use the range notation "2017-2020".
OK.  I'll switch to last year only "2020".